damnit/gui/table.py

```diff
@@ -115,7 +115,11 @@ def set_column_visibility(self, name, visible, for_restore=False):
         deselected. The `for_restore` argument lets you specify which behaviour
         you want.
         """
-        column_index = self.damnit_model.find_column(name, by_title=True)
+        try:
+            column_index = self.damnit_model.find_column(name, by_title=True)
+        except KeyError:
+            log.error("Could not find column %r to set visibility", name)
+            return
```

Comment:
This catch *probably* isn't needed any more with the fix in damnit_ctx.py, but I figured it was easier to leave it in.